crates/ironrdp-async/Cargo.toml

```diff
@@ -11,6 +11,10 @@ authors.workspace = true
 keywords.workspace = true
 categories.workspace = true
 
+[lib]
+doctest = false
+test = false
```

Comment:
Hmm, I am pretty confused - what does this change with `test = false` actually mean in terms of the development process?

As I understand, `cargo test` will now ignore  all unit and integration tests and only something like `cargo test --lib` and `cargo test --bins` will work. However, I have not seen changes in the `workflows`/`xtask` which means tests will be compiled now but not run on ci?

Reply:
Re-posting from slack just so it appears in the GitHub history as well:
`test = false`  means we don’t build any testing binary such as `*unittests` and integration tests in the folder `tests/` for this crate. Instead, we keep the number of binaries to a minimum and only run tests from the testsuite (a single binary).
I didn’t put `test = false` to crates having unit tests such as `ironrdp-pdu` and `ironrdp-graphics` yet, but for crates with no unit tests, it saves us from building and linking another useless `*unittest` binary.

There is nothing to change in the commands we run, `cargo test` will run the testsuite as expected. I only used `cargo build --lib`, `cargo build --bins` and `cargo test --no-run` for benchmarking, we don’t  need those in our build system.